hs/Database/CourseQueries.hs

```diff
@@ -42,16 +57,13 @@ returnCourse lowerStr =
                                                                  TutorialsSession ==. "S"] []
         sqlTutorialsYear   :: [Entity Tutorials]  <- selectList [TutorialsCode ==. courseStr,
                                                                  TutorialsSession ==. "Y"] []
-        
+        let fallSession   = buildSession sqlLecturesFall sqlTutorialsFall
```

Comment:
If these let statements are being moved up here, I don't think we need the second `let`.